### DISCUSSION TITLE
用VS2019基于C++部署最新版本,使用最新的V4模型,CPU识别非常的慢,全部都是默认

### DISCUSSION THREAD

**Discussion by fox37x** [PaddlePaddle/PaddleOCR#12880] [Q&A]
我的电脑是10带i5,如标题所言,我部署后,检测一张小图,进行OCR的时候,要等半分钟左右才能出结果,当我开启enable_mkldnn的时候,非常快,但是没有识别结果.我不知道什么原因导致的,

**Comment by GreatV** [comment]
这个应该和paddle框架支持的cpu指令集有关系，当前（2.6）不支持non-axv512，可以尝试paddle develop版本
